paper/paper.md

```diff
@@ -0,0 +1,281 @@
+  based on the parameters set in the different configuration files.
+Empowered by the Nipype workflow engine, the re-execution of the workflow will resume the
+  processing at any stages a change of parameter occurred.
+
+To ensure reproducibility and maximize re-usability of the tool, the implemented pipelines are encapsulated
+  in a Docker [@merkeldocker:2014] and a Singularity [@Kurtzer2017Singularity:Compute] software image
+  containers, in concordance to the BIDS App framework [@GorgolewskiBIDSMethods:2017].
+This means that the BIDS App of CMP3 can be run on a large diversity of datasets on Linux, MacOSX, Windows computers,
+  and on high performance computing systems (clusters) for large-scale analysis.
+
+Design considerations makes CMP3 not only easy to install and use (as it provides the user with a computing
+  environment in which the pipelines are guarantee to run, and where all dependencies
+  are already installed) on a diversity of multi-modal BIDS datasets, but also efficient in managing and
+  scaling the pipeline execution while recording provenance, and easy to customize and extend it for specific needs.
+At the time EEG is being integrated, CMP3 already provides a collection of interfaces dedicated
+  for this modality that would allow anyone to map the connectivity derived from EEG in
+  the CMP3 framework, as demonstrated in a tutorial notebook.
+
+\textbf{A revisited and extended multi-scale cortical parcellation scheme.}
+CMP3 revisits the multiscale cortical parcellation proposed by [@Cammoun2012:MappingMRI]
+  and its implementation, and extends with new structures including a subdivision for each
+  brain hemisphere of the thalamus into 7 nuclei, the hippocampus into 12 subfields and the brainstem into
+  4 sub-structures (\autoref{fig:parc}).
+
+![\textbf{Creation of the new Lausanne2018 Connectome Parcellation.}
+\label{fig:parc}](Lausanne2018_parcellation_diagram.png)
+
+The parcellation derived from the Desikan-Killiany atlas [@Desikan2006AnInterest] has been
+  made symmetric by projecting the right hemisphere labels to the left hemisphere, matching the
+  boundaries of the projected regions of the left hemisphere to the boundaries of the original regions
+  of the left hemisphere, applying this transformation to the rest of the scales, and saving
+  each parcellation scale in a Freesurfer annotation file.
+After generating the volumetric parcellations from the annotation files, one can now decide or not
+  to perform brainstem parcellation [@Iglesias2015BayesianMRI], hippocampal subfields segmentation [@Iglesias2015AMRI],
+  and / or probabilistic atlas-based segmentation of the thalamic nuclei [@Najdenovska2018In-vivoImaging].
+All segmented structures are combined at the end of process to create the final parcellation nifti image
+  at each scale along with the corresponding label index color mapping file in accordance to the BIDS Derivatives
```

Comment:
I may be wrong but "in accordance with the BIDS" sounds more correct to me than "in accordance to the BIDS".